Assignment 2/Assignment-2/Helper.swift

```diff
@@ -0,0 +1,149 @@
+//
+//  Helper.swift
+//  Assignment-2
+//
+//  Created by Shubham on 01/02/22.
+//
+
+import Foundation
+
+class Helper:Operation{
+func addUser() -> User?{
```

Comment:
These functions should not be part of helper class, A seperate class should be created for these functions, helper class will have helper functions required not the main functions